### PR TITLE
🗃️ Add ClinicianInvite Prisma schema to PI DAS

### PR DESCRIPTION
**Pull request by alyssadsouza**
- added the `ClinicianInvite` Prisma schema and seed data
- added create + read operations
- added the `/clinician-invites` router + swagger jsdocs

### CODE REVIEW

File: apps/pi-das/src/routers/clinicianInvites.ts [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+			participantPreferredName,
+			guardianName,
+			guardianPhoneNumber,
+			guardianEmailAddress,
+			guardianRelationship,
+		});
+		res.status(201).send({ invite });
+	} catch (error) {
+		logger.error(error);
+		res.status(500).send({ error: 'Error creating clincian invite' });
```
Comment (samrichca) [inline apps/pi-das/src/routers/clinicianInvites.ts:176]: clincian invite -> clinician invite (there's a few of these)

Comment (alyssadsouza) [reply apps/pi-das/src/routers/clinicianInvites.ts:176]: whoops 😅 thanks for catching that

File: apps/pi-das/prisma/seed/seed-data.ts [hunk 2]
```diff
@@ -45,4 +45,23 @@ const participants: Prisma.ParticipantCreateInput[] = [
...
-export { participants };
+const clinicianInvites: Prisma.ClinicianInviteCreateInput[] = [
+	{
+		participantFirstName: 'Bruce',
+		participantLastName: 'Wayne',
+		participantEmailAddress: 'bruce.wayne@example.com',
+		participantPhoneNumber: '6475558123',
+	},
+	{
+		participantFirstName: 'Tim',
```
Comment (mistryrn) [inline apps/pi-das/prisma/seed/seed-data.ts:56]: Omg the details 😂 👏

**Review by mistryrn — APPROVED** [review]
Looks good! Let's hold off on merging this one until the PR for #77 (https://github.com/OHCRN/platform/pull/125) is merged into develop

**Review by mistryrn — APPROVED** [review]

**Pull request merged**